### ISSUE TITLE
failed to acquire authentication token (robusto)

### ISSUE THREAD

**Issue by ghost** [EionRobb/icyque#27]
With actual commit  `a10e28e`  I regularly get the following error: `failed to acquire authentication token (robusto)` when starting the client.
Login and authentication works for earlier commits like `513fc16` or sooner.
I am currently on ArchLinux with `pidgin 2.13.0-7`.

**Comment by EionRobb** [comment]
Is your account linked to a phone number?

**Comment by KreMic** [comment]
No it is not. It is the same as it used to be for let's say 15 years. As I wrote login works fine with older commits than the actual one.
Are there any debug informations or tarces to be done which might be useful? Else I would recompile the commit in a proper way.

**Comment by Wertos** [comment]
Failed to acquire authentication token (robusto).

**Comment by Legodev** [comment]
Have the same problem, just updated the icyque because i said, why not and now it brings up the same error message despite the fact it worked with my one year old version. :(

**Comment by H0w4rd** [comment]
For me the commit 513fc16 works as well.

**Comment by TheEMINeM** [comment]
any fix for this guys ?

**Comment by H0w4rd** [comment]
You need to clone it via git, checkout 513fc16 and follow the instructions in README to install it. The commit 513fc16 is still working.

**Comment by loppanloppis** [comment]
This is weird! The current master also gives me the "Failed to acquire authentication token (robusto)." error message. I do receive the code from ICQ though. When I checkout 513fc16 and compile I'm asked for a password when setting up the ICQ account in pidgin. My ICQ account does not have a password, it's a never one for phones only. Any clue what I'm doing wrong here?

**Comment by H0w4rd** [comment]
I don't know. Maybe it's not worth dealing with anymore. The ICQ servers will shut down in a month.

**Comment by loppanloppis** [comment]
True, but my goal is to be able to export my chat history through Pidgin before they shut down.
